houston/environment.py

```diff
@@ -34,5 +34,13 @@ def __getitem__(self, name: str):
         """
         return self.__values[name]
 
+    def __eq__(self, other: 'Environment') -> bool:
+        return self.to_json() == other.to_json()
```

Comment:
Using `to_json` to compare is a bad practice that just happens to work in this instance. I don't think we need to change anything for now, but we'll want to keep a note of this for later. Ideally, Environment should be reimplemented in the same manner as Command and State.